var/spack/repos/builtin/packages/py-region-grower/package.py

```diff
@@ -21,8 +22,10 @@ class PyRegionGrower(PythonPackage):
 
     depends_on('py-click@7.0:', type='run')
     depends_on('py-tqdm@4.28.1:', type='run')
-    depends_on('py-tns@2.2.5:', type='run')
+    depends_on('py-tns@2.4.2:', type='run')
     depends_on('py-voxcell@2.6.3:', type='run')
-    depends_on('py-diameter-synthesis@0.1.7:', type='run')
-    depends_on('py-morphio@2.3.4:', type='run')
+    depends_on('py-diameter-synthesis@0.2.3:', type='run')
+    depends_on('py-morphio@2.7:', type='run')
     depends_on('py-neuroc@0.2.3:', type='run')
+    depends_on('py-neurom@2:2.999', type='run')
+    depends_on('py-morph-tool@2.4.1:', type='run')
```

Comment:
It seems there are still some differences with setup.py:
```
    "attrs>=19.3.0",
    "dask[dataframe, distributed]>=2.15.0",
    "tns>=2.4.0",
    "voxcell>=2.7",
```

Reply:
I added some dask/mpi things, let see!